easybuild/easyconfigs/a/AlphaFold3/AlphaFold3-3.0.0.20250109-foss-2024a-CUDA-12.6.0.eb

```
@@ -0,0 +1,126 @@
+# Thomas Hoffmann, EMBL Heidelberg, structures-it@embl.de, 2024/12
+#
+# DRAFT!
+easyblock = 'PythonBundle'
+
+name = 'AlphaFold3'
+# this commit includes fix for https://github.com/google-deepmind/alphafold3/issues/33# and others...:
+_commit = 'ea040346e10db1759170e723ef263316e64aa768'
+_commit_date = '20250109'
+_version = '3.0.0'
+version = '%s.%s' % (_version, _commit_date)
+versionsuffix = '-CUDA-%(cudaver)s'
+
+homepage = 'https://deepmind.google/technologies/alphafold'
+description = """This package provides an implementation of the inference pipeline of AlphaFold  
+3. See below for how to access the model parameters. You may only use AlphaFold 
+3 model parameters if received directly from Google. Use is subject to these    
+terms of use: 
+https://github.com/google-deepmind/alphafold3/blob/main/WEIGHTS_TERMS_OF_USE.md"""
+
+toolchain = {'name': 'foss', 'version': '2024a'}
+
+github_account = 'google-deepmind'
+use_pip = True
+
+builddependencies = [
+    ('scikit-build', '0.17.6'),
+    ('poetry', '1.8.3'),
+    ('Python-bundle-PyPI', '2024.06'),
+    ('scikit-build-core', '0.10.6'),
+    ('pybind11', '2.13.6'),
+    ('pybind11_abseil', '202402.0'),
+]
+
+dependencies = [
+    ('Python', '3.12.3'),
+    ('CUDA', '12.6.0', '', SYSTEM),
+    ('tqdm', '4.67.0'),
```

```suggestion
    ('tqdm', '4.66.5'),
```
4.67.0 is what's merged in develop.